src/common/sphere_library/CSAssoc.h

```diff
@@ -11,7 +11,7 @@
 
 
 #ifndef OFFSETOF			// stddef.h ?
-	#define OFFSETOF(s,m)   	(int)( (byte *)&(((s *)0)->m) - (byte *)0 )
+	#define OFFSETOF(TYPE, ELEMENT) ((size_t)&(((TYPE *)0)->ELEMENT))
```

Comment:
Did you have the chance to test this change on 32 and 64 bits build, and on platforms different than Mac OS?

Reply:
i have tested it on Windows 64 bit versions let me do the test for 32 bit one. I will get back to you